_posts/2016-07-05-eu-ref.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: UK EU Referendum Update
+categories: future-plans community
+---
+
+The recent EU referendum will undoubtedly have implications for the future of
+UK science. It now seems equally clear that those implications will remain
+unknown for some time to come. No matter what happens next, we want to
+emphasize our ongoing commitment to the OME project and to our community.
+
+It is unlikely that our current funding will be altered by the outcome of the
+referendum or by the apparent upcoming changes to the political landscape. The
+major awards that support OME’s work--from the Wellcome Trust, the BBSRC--are
```

Comment:
from the Wellcome Trust, the BBSRC -> from the Wellcome Trust and the BBSRC

the second the may not be needed.